pkg/storage/image.go

```diff
@@ -616,6 +616,8 @@ func splitDockerDomain(name string) (domain, remainder string) {
 	return
 }
 
+// ResolveNames resolves an image name into a storage image ID or a fully-qualified image name (domain/repo/image:tag).
+// Will only return a slice of fully qualified image names or IDs if err != nil.
```

Comment:
This is the wrong sense.  I'm not worried about a populated slice when `err == nil`, I'm worried about an empty slice when `err == nil`.  I [still like][1]:

> Will only return an empty slice if err == nil.

or

> Will not return an empty slice when err == nil.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1401#discussion_r171719216

Reply:
Updated comment.